rasa/core/migrate.py

```diff
@@ -210,9 +210,9 @@ def _migrate_domain_files(
         backup_location: where to backup all domain files
         out_path: location where to store the migrated files
     """
-    slots = {}
-    forms = {}
-    entities = []
+    slots: Dict[Text, Any] = {}
+    forms: Dict[Text, Any] = {}
+    entities: List[Any] = []
```

Comment:
Unrelated to this PR, just spotted something wrong in L264, the default in dict.get should be `[]`, instead of `{}`:
```
entities.extend(original_content.get(KEY_ENTITIES, []))
```